assets/src/edit-story/masks/frame.js

```diff
@@ -107,28 +138,16 @@ function WithDropTarget({ element, children, hover }) {
         // reaching the frame through zIndex
         active={activeDropTargetId === element.id}
       >
-        {/** Suble indicator that the element has a drop target */}
-        <DropTargetPath
-          {...pathProps}
-          strokeWidth="4"
-          style={
-            (hover && !draggingResource) ||
-            (Boolean(draggingResource) &&
-              isDropSource(draggingResource.type) &&
-              draggingResource !== resource)
-              ? { opacity: 1 }
-              : {}
-          }
-        />
-        {/** Drop target shown when an element is in the drop target area  */}
+        {/** Suble indicator that the element is a drop target */}
         <DropTargetPath
-          ref={pathRef}
           {...pathProps}
-          strokeWidth="48"
-          active={activeDropTargetId === element.id}
+          strokeWidth="3"
+          style={{ opacity: hasThinOutline ? 1 : 0 }}
         />
+        {/** Drop target snap border when an element is in the drop target area */}
+        <DropTargetPath ref={pathRef} strokeWidth="48" {...pathProps} />
```

Comment:
The blue border is still shown in the Helper btw -- probably that should get updated, too? (Not in this PR though!)

The drop-targeting feels a bit "rough" and unexpected to me now so would be good if Sam could also test it, however, this can be done in UAT.